build/assets/tuned/openshift-node-network-latency

```diff
@@ -0,0 +1,52 @@
+[main]
+summary=Openshift node optimized for deterministic performance at the cost of increased power consumption, focused on low latency network performance
+include=openshift-node
+
+[cpu]
+# https://github.com/redhat-performance/tuned/blob/master/profiles/latency-performance/tuned.conf
+# https://github.com/redhat-performance/tuned/blob/master/profiles/network-latency/tuned.conf
+force_latency=cstate.id:1|3
```

Comment:
there might be an issue with this format of setting latency  (using the OR operand) , investigating this but worth checking if tuned pod doesn't have any errors with that here .

Reply:
We can fix it under the separate PR